ruby/ql/consistency-queries/DataFlowConsistency.ql

```diff
@@ -35,11 +35,17 @@ private module Input implements InputSig<RubyDataFlow> {
       n.asExpr() = arg
     )
   }
+
+  predicate multipleArgumentCallExclude(ArgumentNode arg, DataFlowCall call) {
+    // An argument such as `x` in `if not x then ...` has two successors (and hence
+    // two calls); one for each Boolean outcome of `x`.
```

Comment:
Maybe I just don't know enough about the Ruby modeling, but the `x` in `if not x then ...` doesn't look like an argument to me, and I don't see any calls, so this comment reads a bit cryptic.